devtools/dump_devinfo.py

```diff
@@ -233,6 +233,12 @@ async def handle_device(
     type=bool,
     help="Set flag if the device encryption uses https.",
 )
+@click.option(
+    "--timeout",
+    required=False,
+    default=15,
+    help="Timeout for queries.",
+)
```

Comment:
Worth having a separate PR for timeout change? Your call.

Reply:
Agreed, that's done in https://github.com/python-kasa/python-kasa/pull/1381. Will also separate the miscellaneous `dump_devinfo` fixes in this PR into a separate PR once 1381 is merged.

Reply:
Okay, merged it :+1:

Reply:
Misc fixes extracted into https://github.com/python-kasa/python-kasa/pull/1382